content/en/docs/reference/command-line-tools-reference/feature-gates/consistent-list-from-cache.md

```diff
@@ -9,6 +9,15 @@ stages:
   - stage: alpha
     defaultValue: false
     fromVersion: "1.28"
+    toVersion: "1.30"
+  - stage: beta
+    defaultValue: true
+    fromVersion: "1.31"
 ---
-Allow the API server to serve consistent lists from cache.
+Enhance Kubernetes API server performance by serving consistent list requests
+directly from its watch cache, improving scalability and response times.
+This feature requires a newer etcd version (v3.4.33+ or v3.5.13+) and ensures
```

Comment:
I've guessed at what you meant. Is this right?
```suggestion
Unless you explicitly disable this feature gate, the kube-apiserver component requires
that etcd is compatible (v3.4.33+, v3.5.13+, or any version v3.6.0 or later).
Serving collections from the watch cache ensures
```

Reply:
No, kube-apiserver will work either way. This feature  becomes default as it goes to beta, we cannot break apiserver. Enablement of the feature requires both feature flag, and etcd to be of sufficient version. Kube-apiserver will check etcd version and disable the feature if it detects older etcd version.

Reply:
OK; please document that. Don't write “this feature”, though; explain the _behavior_ not the enhancement that made it possible.

Reply:
Done